packages/mdc-menu-surface/foundation.js

```diff
@@ -124,8 +124,8 @@ class MDCMenuSurfaceFoundation extends MDCFoundation {
     this.animationRequestId_ = 0;
     /** @private {!{ width: number, height: number }} */
     this.dimensions_;
-    /** @private {MenuSurfaceCorner} */
-    this.anchorCorner_ = MenuSurfaceCorner.TOP_START;
+    /** @private {Corner} */
```

Comment:
You might want to double check with @bwobrien if these type annotations need to be marked non-null - e.g., `{!Corner}`.

I realize `Corner` is an `@enum`, which is basically just an alias for `number` which is non-null by default, but I vaguely recall something in the Google JS style guidelines and/or Closure Compiler docs about how you should always mark them explicitly non-null regardless.

Ditto elsewhere in this PR.

Reply:
Style guidelines say to always include either `?` or `!` for enums.